data.py

```diff
@@ -168,4 +168,5 @@
 You have access to the chat history of the current chat.
 Use the conversation history as context for your prompts.
 If asked about what happened in earlier messages, use the chat history given to you.
+Do not under any circumstances say 'Generating...', no exceptions.
```

Comment:
Did this happen in testing, or why add this specifically?

Reply:
Check the gpt code for why this is